COPYRIGHT.txt

```diff
@@ -87,9 +87,9 @@ Copyright: 2002, Google Inc.
 License: Apache-2.0
 
 Files: ./platform/android/power_android.cpp
+ ./platform/freedesktop/power_x11.cpp
```

Comment:
Just being a little nitpicky here, but if we're renaming the `x11` platform to `freedesktop`, shouldn't the filenames be changed too?:

Reply:
Yes, AFAIK, there is nothing X specific in that file, it should be renamed.